src/torch_ppr/utils.py

```diff
@@ -197,15 +215,15 @@ def prepare_page_rank_adjacency(
     adj = edge_index_to_sparse_matrix(edge_index=edge_index, num_nodes=num_nodes)
     # symmetrize
     adj = adj + adj.t()
+    if add_identity:
+        adj = adj + sparse_diagonal(torch.ones(adj.shape[0], dtype=adj.dtype, device=adj.device))
+
     # TODO: should we add an identity matrix here?
```

Comment:
can this TODO be removed?

Reply:
yes. good catch.

Reply:
https://github.com/mberr/torch-ppr/pull/20/commits/44f4f2e8132ab4cad85890d4489176696b9883dc